app/assets/javascripts/discourse/tests/acceptance/bookmarks-test.js

```diff
@@ -162,6 +167,9 @@ acceptance("Bookmarking", function (needs) {
     await selectKit(".bookmark-option-selector").expand();
     await selectKit(".bookmark-option-selector").selectRowByValue(1);
     await click("#save-bookmark");
+
+    assert.equal(User.current().bookmark_auto_delete_preference, "1");
```

Comment:
I'm wondering if we need to assert for this when the following assertions after we open the edit bookmark modal is asserting that the options panel is expanded by default.

Reply:
I think so. This checks that the user preference was saved for the current user session and the next assert checks if the options panel is expanded only when editing the bookmarks.